### PR TITLE
Enable Recording in Daily Video Calls

### PR DESCRIPTION
**Pull request by lunchpaillola**
Added the ability to record calls to the Daily video integration and updated the ReadMe. Any [Daily scale plan](https://www.daily.co/pricing) user can record calls. To enable you would need to update the Daily_Scale_Plan environment variable to "enable" which will allow recording and saving recordings locally.

Here's a screenshot of what that looks like
CleanShot 2021-11-05 at 22 36 30

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -292,6 +292,7 @@ Contributions are what make the open source community such an amazing place to b
  2. From within your dashboard, go to the [developers](https://dashboard.daily.co/developers) tab.
  3. Copy your API key.
  4. Now paste the API key to your .env file into the `DAILY_API_KEY` field in your .env file.
+ 5. If you have a [Daily Scale Plan](https://www.daily.co/pricing)  can also enable the ability to record Daily video meetings. To do so, set the `DAILY_SCALE_PLAN` environment variable to `enable`
```
Comment (baileypumfleet) [inline README.md:295]: Could we tell people to set the variable to `true` for consistency? I don't think it should affect the rest of this code if people set it to true

**Review by pumfleet — CHANGES_REQUESTED** [review]

File: lib/integrations/Daily/DailyVideoApiAdapter.ts [hunk 2]
```diff
@@ -97,6 +97,23 @@ const DailyVideoApiAdapter = (credential: Credential): VideoApiAdapter => {
     // added a 1 hour buffer for room expiration and room entry
     const exp = Math.round(new Date(event.endTime).getTime() / 1000) + 60 * 60;
     const nbf = Math.round(new Date(event.startTime).getTime() / 1000) - 60 * 60;
+    const scalePlan = process.env.DAILY_SCALE_PLAN;
```
Comment (PeerRich) [inline lib/integrations/Daily/DailyVideoApiAdapter.ts:100]: this looks like it's not checking for "enable"

we should just check for boolean instead (true)

**Review by PeerRich — APPROVED** [review]

File: lib/integrations/Daily/DailyVideoApiAdapter.ts (hunk 2 above)
Comment (lunchpaillola) [reply lib/integrations/Daily/DailyVideoApiAdapter.ts:100]: I updated it to a true

File: README.md (hunk 1 above)
Comment (lunchpaillola) [reply README.md]: Sure, can. It's updated thanks!

**Review by pumfleet — APPROVED** [review]

**Comment by baileypumfleet** [comment]
Looks good to go as far as I can see :) Unfortunately your PR is locked, so I can't update the branch, so if you could do it and drop me a message on Slack, I'll get it merged right away

**Pull request merged**